### PR TITLE
bus_mapping: add `jump` and `jumpi` opcode

### PR DESCRIPTION
**Pull request by scroll-dev**

### CODE REVIEW

File: bus-mapping/src/evm/opcodes/jumpi.rs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        eth_types::Word,
+        evm::StackAddress,
+        mock,
+    };
+    use pretty_assertions::assert_eq;
+
+    #[test]
+    fn jumpi_opcode_impl() -> Result<(), Error> {
+        let condition = 1;
+        let destination = 35;
```
Comment (han0110) [inline bus-mapping/src/evm/opcodes/jumpi.rs:54]: Tho it doesn't affect the zero chunk write in line 63-65, but maybe with a larger value or at least `68` makes more sense?

**Review by han0110 — APPROVED** [review]
LGTM! Only one unimportant nitpick.

File: bus-mapping/src/evm/opcodes/jumpi.rs (hunk 1 above)
Comment (HAOYUatHZ) [reply bus-mapping/src/evm/opcodes/jumpi.rs]: make sense! 9c97d140efc5b67bf0fd4905bde276d8f6d201c4

**Review by CPerezz — APPROVED** [review]
LGTM

**Pull request merged**